doc/nrf/ug_matter_overview_device_types.rst

```diff
@@ -0,0 +1,210 @@
+.. _ug_matter_device_types:
+
+Matter device types
+###################
+
+.. contents::
+   :local:
+   :depth: 2
+
+A Matter device type is an officially defined collection of requirements for one or more endpoints.
+Device types are intended to ensure interoperability of different device brands on the market.
+
+All device types are defined in the Device Library specification, which is available to `Connectivity Standards Alliance`_ members.
+Each device type definition is composed of the following elements:
+
+* Device type ID
+* Device type revision
+* One or more mandatory clusters, including each cluster's minimum revision
+
+The device type definition in the Device Library specification can change over time.
+The changes are tracked using the Device Type Revision value, which is initially set to ``1``.
+Changes do not modify how the device works, but only improve its functionality.
+
+A device type can require other device types for its composition, making it a *composed device type*.
+
+Device type overview
+********************
+
+The following tables list the application device types that are supported in the Matter specification.
+
+* The description for each device type is taken from the Device Library specification.
+* The state of the device specifies whether a device type can be subject to certification.
+  The `Provisional` state indicates that the device type is not yet ready for implementation.
```

Comment:
I would try to rephrase it a little bit to better describe the situation for a user. Provisional state doesn't describe the implementation state, but rather certification state. It means that implementation is most likely ready, but this device type didn't get enough coverage (to small amount of people tested it) to claim it is certified. Would something like this suite you:
```suggestion
  The `Provisional` state indicates that the device type implementation was not yet fully tested and certified, however implementation may be ready and can be used by a user on his own risk.
```

Reply:
Rephrased slightly.